### ISSUE TITLE
Tex file not outputted with .u3d file export

### ISSUE THREAD

**Issue by IRS001** [cnr-isti-vclab/meshlab#724]
#### Check all platforms you experienced the issue (change to `[x]`)
- [x ] Windows 10
- [ ] Mac OS
- [ ] Linux
- [ ] Every platform

#### What type of issue is?
- [x ] bug or unexpected behavior
- [ ] crash
- [ ] feature request
- [ ] other (specify...)

#### Describe your issue
When I export a .u3d file from version 2020.06, from what I have read, a .tex file relating to this document is normally also exported.   This is not happening, with only the .u3d file exported
...

**Comment by alemuntoni** [comment]
Duplicate of #495

**Comment by IRS001** [comment]
Hello Alessandro, My problem seems to the reverse of #495.  I’m getting the u3d file exported, but not the tex file.


Thanks,
Steve

**Comment by alemuntoni** [comment]
Could you please try the [last built meshlab](https://github.com/cnr-isti-vclab/meshlab/actions/runs/139847931) after #726 was merged? In case the problem persists, please provide some minimum details to get the problem reproducible (example: the mesh that you are trying to export).

**Comment by IRS001** [comment]
Hello Alessandro,
    This version is still only outputting the u3d file, and no TEX file.

Attached is the stl file I generated from AutoCAD.  Hope that helps,

Thanks,
Steve

**Comment by alemuntoni** [comment]
You need to reply on github (and not via email), or the attachment won't be visible.

**Comment by IRS001** [comment]
STL file is attached to this upload.
Thanks.
[Sand Strainer 3D.zip](https://github.com/cnr-isti-vclab/meshlab/files/4822109/Sand.Strainer.3D.zip)

**Comment by alemuntoni** [comment]
Hi @IRS001, sorry for the late reply.
It seems to work with 2020.12, could you please try it?
